core/src/main/java/io/micronaut/core/beans/internal/DefaultBeanWrapper.java

```diff
@@ -31,7 +33,7 @@
  * @since 1.1
  */
 @Internal
-final class DefaultBeanWrapper<T> implements BeanWrapper<T> {
+public final class DefaultBeanWrapper<T> implements BeanWrapper<T> {
```

Comment:
seems weird we have to make these classes public when they were already package private?

Reply:
Yes I don't like it either. It was necessary to move them to a "internal" package. Should I move them back out again?

Reply:
yeah for me I don't understand why this change is necessary, is it a JPM requirement? If it isn't then they shouldn't moved. They fact they are internal is already expressed by the fact they are package private